extensions/cornerstone/src/commandsModule.ts

```diff
@@ -565,17 +522,23 @@ function commandsModule({
         (currentIndex + direction + viewportIds.length) % viewportIds.length;
       viewportGridService.setActiveViewportId(viewportIds[nextViewportIndex] as string);
     },
-
-    toggleImageSliceSync: ({ toggledState }) => {
-      toggleImageSliceSync({
-        servicesManager,
-        toggledState,
-      });
+    toggleSynchronizer: ({ type, viewports }) => {
+      if (type === 'imageSlice') {
+        toggleImageSliceSync({
+          servicesManager,
+          viewports,
+        });
+      } else if (type === 'voi') {
+        toggleVOISliceSync({
+          servicesManager,
+          viewports,
+        });
```

Comment:
Is it possible to call the method with syncId directly, and then call the setEnable method on the sync - that would presuppose the sync already existed, but I think that should be an option.